### PR TITLE
SOLACE: Enhance Chat Bot UI

### PR DESCRIPTION
**Pull request by pankaj3108**
Enhance Chat Bot UI

### CODE REVIEW

File: src/components/ChatBot/ChatBot.tsx [hunk 1]
```diff
@@ -78,20 +83,32 @@ const Chatbot: React.FC<ChatBotProps> = ({selectionRef, documentId}) => {
...
+        <TextareaAutosize
+          onChange={e => handleTextAreaChange(e)}
           value={userResponse}
-        ></input>
-        <button>
-          <i className="far fa-paper-plane"></i>
-        </button>
+          placeholder="Start asking questions..."
+        ></TextareaAutosize>
+        {/* <img src={sendLogo} alt="sendLogo" onClick={e => handleSubmit(e)}/> */}
```
Comment (rishup132) [inline src/components/ChatBot/ChatBot.tsx:108]: Remove this.

File: src/components/ChatBot/ChatBot.scss [hunk 2]
```diff
@@ -44,6 +41,10 @@
     }
 }
 
+.button {
+  border-radius: 50%;
+}
+
```
Comment (rishup132) [inline src/components/ChatBot/ChatBot.scss:47]: Where are we using it?

File: src/components/ChatBot/ChatBot.scss [hunk 3]
```diff
@@ -33,7 +30,7 @@
         button {
             background: #e5dfdf;
             color: #393e46;
-            border-style: none;
+            margin-left: 15px;
             @include form-component-style;
```
Comment (rishup132) [inline src/components/ChatBot/ChatBot.scss:34]: Are we using it?

File: src/components/ChatBot/ChatBot.scss [hunk 4]
```diff
@@ -1,5 +1,4 @@
 @mixin form-component-style {
-    border-radius: 10px;
     padding: 0.5em;
```
Comment (rishup132) [inline src/components/ChatBot/ChatBot.scss:2]: Delete this file as it is not used.

File: src/components/ChatBot/ChatBot.tsx (hunk 1 above)
Comment (pankaj3108) [reply src/components/ChatBot/ChatBot.tsx]: Removed the file.

File: src/components/ChatBot/ChatBot.scss (hunk 2 above)
Comment (pankaj3108) [reply src/components/ChatBot/ChatBot.scss]: Removed the file.

File: src/components/ChatBot/ChatBot.scss (hunk 3 above)
Comment (pankaj3108) [reply src/components/ChatBot/ChatBot.scss]: Removed the file.

File: src/components/ChatBot/ChatBot.scss (hunk 4 above)
Comment (pankaj3108) [reply src/components/ChatBot/ChatBot.scss]: Removed the file.

**Pull request merged**